lang/ilias_en.lang

```diff
@@ -1372,6 +1372,7 @@ assessment#:#tst_count_correct_solutions#:#Only Correct and Complete Answers Sco
 assessment#:#tst_count_correct_solutions_desc#:#Participants score either the maximum number of points for a fully correct and complete answer or 0 points in every other case. This is also valid for questions that define points for partial answers.
 assessment#:#tst_count_partial_solutions#:#Incomplete or Partly Wrong Answers Score Points
 assessment#:#tst_count_partial_solutions_desc#:#When a question defines points for partial answers, points for correctly provided partial answers are added up: Participants can score points for incomplete or partly wrong answers on these questions.
+assessment#:#tst_current_pass_no_longer_valid#:#Your current Test Pass is no longer valid. <br>It was probably already completed or finished by a Tutor.
```

Comment:
According to our English expert Chris Potter, the term 'pass' should only be used for a successful test run (I still would like to streamline this until 8.0 stable). The use of 'pass' in this lang var shows the problem. The test run is not even completed but called pass. That's confusing for a native speaker. Better would be: 'Your current test run is no longer valid. <br>It was probably completed already or finished by a tutor.' And neither 'test run' nor 'Tutor' needs capital letters. 'Tutor' in capital letters would refer to the role 'Tutor', while 'tutor' is a general term for those people who are tutoring...